content/en/cloud_cost_management/recommendations/custom_recommendations.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Custom Recommendations
+description: Learn how to customize and manage custom recommendations to fit your business needs.
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+- link: "/cloud_cost_management/recommendations"
+  tag: "Documentation"
+  text: "Cloud Cost Recommendations"
+
+---
+
+{{< callout url="#" btn_hidden="true" header="false">}}
+  Custom Recommendations is in Preview.
+{{< /callout >}}
+
+## Overview
+
+{{< img src="cloud_cost/.png" alt="Configure custom recommendations from the Cloud Cost Settings page." style="width:80%;" >}}
+
+While [CCM Recommendations][1] are designed to be informative and actionable from the start, you can customize recommendations that you receive to fit your specific business use cases and needs. Set your own metric thresholds and evaluation time frames to reduce unnecessary alerts and focus on the recommendations that matter most.
```

Comment:
I think the "While" makes it sound like these two are in competition, but is this more of an additional feature to CCM Recommendations?

```suggestion
[CCM Recommendations][1] are designed to be informative and actionable from the start. You can customize these recommendations to fit your specific business use cases and needs. Set your own metric thresholds and evaluation time frames to reduce unnecessary alerts and focus on the recommendations that matter most.
```